scripts/github-actions/tests-e2e-samples

```diff
@@ -33,6 +33,6 @@ MOCK_GROUP_EMAIL="test@group.com"
 RUN_E2E=1 \
 E2E_KUBE_TARGET=envtest \
 E2E_GCP_TARGET=mock \
-KCC_USE_DIRECT_RECONCILERS="ComputeForwardingRule" \
+KCC_USE_DIRECT_RECONCILERS="ComputeForwardingRule,SpannerInstance" \
```

Comment:
We no longer need to flip the KCC_USE_DIRECT_RECONCILERS. If the presubmit passes (fixture passes), we can  assume the direct approach for sample is good as well. For fixtures, if we want to support direct approach, we can use the direct alpha annotation.